packages/calcite-components/src/demos/dialog.html

```diff
@@ -0,0 +1,1795 @@
+      }
+    </style>
+
+    <script src="./_assets/head.js"></script>
+  </head>
+
+  <body>
+    <demo-dom-swapper>
+      <h1 style="margin: 0 auto; text-align: center">Dialog</h1>
+
+      <!-- Sizes -->
+      <div class="parent">
+        <div class="child right-aligned-text"></div>
+        <div class="child">Small</div>
+        <div class="child">Medium</div>
+        <div class="child">Large</div>
+      </div>
+
+      <!--
+      **************************************************
+      * SMALL
+      **************************************************
+    -->
+      <div class="parent">
+        <div class="child right-aligned-text">WidthScale S</div>
+
+        <!-- small widthScale & scale -->
+        <div class="child">
+          <calcite-dialog
+            id="js-dialog-s-s"
+            width-scale="s"
+            scale="s"
+            heading="Small widthScale and small scale dialog"
+            description="My description"
+          >
+            <p>
+              The small dialog is perfect for short confirmation dialogs or very compact interfaces with few elements.
+            </p>
+            <calcite-button
+              slot="footer-start"
+              width="full"
+              scale="s"
+              kind="neutral"
+              appearance="outline"
+              icon-start="chevron-left"
+              >Back</calcite-button
+            >
+            <calcite-button slot="footer-end" width="full" scale="s" appearance="outline">Cancel</calcite-button>
+            <calcite-button slot="footer-end" width="full" scale="s">Save</calcite-button>
+          </calcite-dialog>
+
+          <calcite-button data-dialog-ref="js-dialog-s-s" appearance="outline" scale="s"> Scale Small </calcite-button>
+        </div>
+
+        <!-- small widthScale & medium scale -->
+        <div class="child">
+          <calcite-dialog
+            heading="Small widthScale and medium scale dialog"
+            description="My description"
+            id="js-dialog-s-m"
+            width-scale="s"
+            scale="m"
+          >
+            <p>
+              The small dialog is perfect for short confirmation dialogs or very compact interfaces with few elements.
+            </p>
+            <calcite-button
+              slot="footer-start"
+              width="full"
+              scale="m"
+              kind="neutral"
+              appearance="outline"
+              icon-start="chevron-left"
+              >Back</calcite-button
+            >
+            <calcite-button slot="footer-end" width="full" scale="m" appearance="outline">Cancel</calcite-button>
+            <calcite-button slot="footer-end" width="full" scale="m">Save</calcite-button>
+          </calcite-dialog>
+
+          <calcite-button data-dialog-ref="js-dialog-s-m" appearance="outline" scale="s"> Scale Medium </calcite-button>
+        </div>
+
+        <!-- small widthScale & large scale -->
```

Comment:
Nitpick: can we drop these comments and rely on the dialog headings and `.child` text elements?

Reply:
Yeah, they are so redundant